packages/lib-classifier/webpack.dev.js

```diff
@@ -52,10 +47,13 @@ module.exports = {
   },
   plugins: [
     EnvironmentWebpackPlugin,
-    HtmlWebpackPluginConfig,
-    WorkboxPluginConfig
+    HtmlWebpackPluginConfig
   ],
   resolve: {
+    alias: {
+      // adjust this path as needed depending on where your webpack config is
+      'styled-components': path.resolve('./node_modules/styled-components')
```

Comment:
Won't this resolve to the version of styled-components in this library, not the consuming app?

Reply:
This is the dev version, so I think it's fine.

For production, though, should we bite the bullet and hoist shared modules like `styled-components` and `react` to the top-level `node_modules` directory?

Reply:
Yeah it's for the dev version locally to this lib. We definitely need to move to hoisting at some point.